### DISCUSSION TITLE
No response after startup, black screen, sometime. Reinsert the power cord will be fine.

### DISCUSSION THREAD

**Discussion by nimsun08** [osy86/HaC-Mini#1011] [Q&A]
No response after startup, black screen, sometime. Reinsert the power cord will be fine.

Configuration
- HaC Mini version: 3.1.4
- OSX version : 11.6.1
- NUC model : NUC8i7HVK:
- Boot SSD model and protocol : Samsung 980 1TB, Nvme
- First DIMM model : Asgard DDR4-3200 8GB]: 
- Second DIMM model Asgard DDR4-3200 8GB: 
- Display port(s) used HDMI
- Display resolution(s) 4K
- Any TB3 devices: No
- Any USB devices: No
- Any Wi-Fi/Bluetooth devices : BCM94360CS2

**Comment by Medizinmann-MD** [comment]
I have seen this or similar behavior (sporadically) with my main monitor (AOC Agon AG271QG) on HDMI when starting the HaC and the monitor in suspend mode - activating the monitor first/before boot helps.

**Comment by osy** [comment]
Yeah this is a known issue. It’s pretty annoying.

**Comment by nimsun08** [comment]
Config–NVRAM–WriteFlash, i have turn it off for half a month. it seems be
good so far.

**Comment by aareet** [comment]
I turned off hibernation and sleep, that prevents this issue as well, but I will try the NVRAM fix from nimsun08 as that would be preferable
